src/constants/ConstantsOwnersAction.js

```diff
@@ -0,0 +1,5 @@
+export default {
+  MESSAGE_ACCEPT_DELETE: (name) =>
+    `Are you sure, that you wanna delete ${name} from owners?`,
```

Comment:
For `Are you sure, that you wanna delete` and `from owners` need to create constants too

Reply:
+